contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1302,36 +1306,23 @@ Taints:             node-role.kubernetes.io/infra:NoSchedule
 +
 Do not use `[...]`, `<snip>`, or any other variant.
 
-* Do not use `jq` in commands (unless it is truly required), because this requires users to install the `jq` tool. Oftentimes, the same or similar result can be accomplished using `jsonpath` for `oc` commands.
-+
+[[jq-commands]]
+==== Commands with jq
+Do not use `jq` in commands (unless it is truly required), because this requires users to install the `jq` tool. Oftentimes, the same or similar result can be accomplished using `jsonpath` for `oc` commands.
+
 For example, this command that uses `jq`:
-+
+
 ----
 $ oc get clusterversion -o json|jq ".items[0].spec"
 ----
-+
+
 can be updated to use `jsonpath` instead:
-+
-----
-$ oc get clusterversion -o jsonpath='{.items[0].spec}{"\n"}'
-----
 
-* For Bash "here" documents use `[source,terminal]`, such as the following example:
-+
-....
-[source,terminal]
 ----
-$ cat <<EOF| oc create -f -
-apiVersion: v1
-kind: Pod
-metadata:
-  name: mlistener
-  labels:
-    app: multicast-verify
-EOF
+$ oc get clusterversion -o jsonpath='{.items[0].spec}{"\n"}'
 ----
-....
 
+//this contradicts the earlier guidance on line 1006
```

Comment:
Then we should remove this section - the guidance in line 1006 has much higher usage and, IMO, is more appropriate.

Reply:
Thank you; updated.